WordPress/Classes/ViewRelated/Reader/WPStyleGuide+ReaderComments.swift

```diff
@@ -3,10 +3,15 @@ import WordPressShared
 
 extension WPStyleGuide {
 
-    public class func defaultSearchBarTextAttributes(_ color: UIColor) -> [String: AnyObject] {
+    @objc class func defaultSearchBarTextAttributes(_ color: UIColor) -> [String: Any] {
+        let attributes = defaultSearchBarTextAttributesSwifted(color)
+        return NSAttributedStringKey.convertToRaw(attributes: attributes)
+    }
+
+    class func defaultSearchBarTextAttributesSwifted(_ color: UIColor) -> [NSAttributedStringKey: Any] {
```

Comment:
Maybe we can keep just this one, mark it `@objc` and drop the Swifted from the name. Then use the converters when necessary?

Reply:
Let's keep it (for a little bit!), since it's used in a few spots, and may make things (slightly) easier. The method above is (ALSO) used in Swift, because few of the UITextView properties don't support, yet, NSAttributedStringKey.

🔪